### PR TITLE
Killing driver's process tree instead of single process

### PR DESCRIPTION
**Pull request by yaryabu**
1. Right now driver's process receives SIGKILL and all of it's children stay launched, this PR fixes it
2. Fixed problem with zombie processes hanging forever

### CODE REVIEW

**Comment by vania-pooh** [comment]
Summon @aandryashin.

File: service/driver.go [hunk 1]
```diff
@@ -68,12 +70,23 @@ func (d *Driver) StartWithCancel() (*StartedService, error) {
...
+		killProcessTree(child.Pid, requestId)
+	}
+	log.Printf("[%d] [TERMINATING_PROCESS] [%d]\n", requestId, pid)
+	proc.Kill()
+}
+
+func (d *Driver) stopProcess(cmd *exec.Cmd) {
+	log.Printf("[%d] [TERMINATING_PROCESS_TREE] [%d]\n", d.RequestId, cmd.Process.Pid)
+	killProcessTree(int32(cmd.Process.Pid), d.RequestId)
+	go cmd.Wait()
```
Comment (lanwen) [inline service/driver.go]: Why wait in goroutine?

Comment (yaryabu) [reply service/driver.go]: If we don't cmd.Wait(), processes continue to live on as zombies until selenoid itself is shut down. 
I'm new to Go, so I'm not sure if this has to be a goroutine, I just assume that it won't block the main thread if something goes wrong with process killing and wait.

**Comment by vania-pooh** [comment]
@yaryabu after some discussion - we decided that it would be great to move this logic to a separate binary (which could be then reused not only with Selenium). Then in `browsers.json` user will specify a modified command including this new binary and the rest of arguments. We will be happy to publish this to a separate repository in Aerokube organization and provide any help with code \ documentation. Please copy the code to a separate repository and send us the link.

**Pull request closed without merging**